src/plrust.rs

```diff
@@ -43,6 +43,23 @@ pub(crate) unsafe fn unload_function(fn_oid: pg_sys::Oid) {
     })
 }
 
+#[tracing::instrument(level = "debug")]
+pub(crate) unsafe fn recompile_missing_shared_object(fn_oid: pg_sys::Oid) -> eyre::Result<()> {
+    let db_oid = unsafe { MyDatabaseId };
+    let mut shared_object_name = crate_name(db_oid, fn_oid);
+    shared_object_name.push_str(DLL_SUFFIX);
+    let shared_library = gucs::work_dir().join(&shared_object_name);
+    if !shared_library.exists() && gucs::auto_recompile() {
+        tracing::trace!(
+            "Shared object {shared_object} is not found and plrust.auto_recompile is enabled, recompiling {shared_object}",
+            shared_object = shared_object_name
```

Comment:
This is going to some effort to avoid allowing the formatting macro to simply capture the function.
```suggestion
    let mut shared_object = crate_name(db_oid, fn_oid);
    shared_object.push_str(DLL_SUFFIX);
    let shared_library = gucs::work_dir().join(&shared_object);
    if !shared_library.exists() && gucs::auto_recompile() {
        tracing::trace!(
            "Shared object {shared_object} is not found and plrust.auto_recompile is enabled, recompiling {shared_object}",
```